ckks/params.go

```diff
@@ -157,14 +313,21 @@ type ParametersLiteral struct {
 	Sigma    float64 // Gaussian sampling variance
 	LogSlots int
 	Scale    float64
+	RingType ring.Type
 }
 
 // DefaultParams is a set of default CKKS parameters ensuring 128 bit security in a classic setting.
 var DefaultParams = []ParametersLiteral{PN12QP109, PN13QP218, PN14QP438, PN15QP880, PN16QP1761}
 
+// DefaultParamsRCKKS is a set of default RCKKS parameters ensuring 128 bit security in a classic setting.
+var DefaultParamsRCKKS = []ParametersLiteral{PN12QP109RCKKS, PN13QP218RCKKS, PN14QP438RCKKS, PN15QP880RCKKS, PN16QP1761RCKKS}
```

Comment:
should be `DefaultRCKKSParams` for consistency with `DefaultPostQuantumParams`.

Reply:
Additionally, we should probably avoid the "RCKKS" which we kind of invented as an informal description of the scheme.